redis/commands/graph/query_result.py

```diff
@@ -325,6 +329,10 @@ def _get_stat(self, stat):
     def labels_added(self):
         return self._get_stat(LABELS_ADDED)
 
+    @property
+    def labels_removed(self):
+        return self._get_stat(LABELS_REMOVED)
```

Comment:
Even though they're "gimmes" we should have unit tests, covering each property - this way we don't break them in the future.